hcloud/load_balancers/client.py

```diff
@@ -83,6 +84,17 @@ def __init__(self, client: LoadBalancersClient, data: dict, complete: bool = Tru
                     tmp_target.use_private_ip = target["use_private_ip"]
                 elif target["type"] == "ip":
                     tmp_target.ip = LoadBalancerTargetIP(ip=target["ip"]["ip"])
+
+                target_health_status = target.get("health_status")
```

Comment:
Is there a difference between `target.get("health_status")` and `target["health_status"]` (which most lines above are using)?

Reply:
- `target["health_status"]` might raise a KeyError if the key is not present in the dict. We should then check if the key is inside with `if "health_status" in target:`
- `target.get("health_status")` returns `None` if the value is not set, or can return the default if provided `target.get("health_status", "my default value")`